mache-chroniclemap/src/test/java/com/excelian/mache/chroniclemap/ChronicleMapMacheProvisionerShould.java

```diff
@@ -0,0 +1,88 @@
+package com.excelian.mache.chroniclemap;
+
+import com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.ChronicleMapBuilderConfig;
+import com.excelian.mache.core.HashMapCacheLoader;
+import net.openhft.chronicle.map.ChronicleMapBuilder;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.util.List;
+
+import static com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.chronicleMap;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.internal.util.reflection.Whitebox.getInternalState;
+
+public class ChronicleMapMacheProvisionerShould {
+
```

Comment:
Throwing in my 2c - a little bit of playing devils advocate and an initial distaste (which receded writing this!). I like all test classes ending Test as it makes navigation easier (I suppose *Test + *Should would be ok but ideally one or the other) and I also think if your looking at this from a linguistic perspective then the verb should come first e.g. "GetShouldReturn404ForMissingKey" or BDD style "Given_MissingKey_When_GetCalled_Then_Return404" underscores or camelcase aside that reads nicer to me than "Return_404MissingKey_When_GetCalled" (or similar) and a failure of MyWebsiteHostShould.Return_404MissingKey_When_GetCalled.

I would say its far more important though that tests read like sentences one way or another and that we employ a consistent approach.